index.js

```diff
@@ -68,8 +68,9 @@ cli.on('require', function(name) {
   log.info('Requiring external module', ansi.magenta(name));
 });
 
-cli.on('requireFail', function(name) {
-  log.warn(ansi.yellow('Failed to load external module'), ansi.magenta(name));
+cli.on('requireFail', function(name, error) {
+  var message = name + (error ? ' [' + error + ']' : '');
```

Comment:
Since `error` is an error object, this code outputs too many lines including stack trace. Isn't it enough error message? And I think it's better that the error message is outputted to next line.

Reply:
Fo me this message:
```
Failed to load external module @babel/register (Error: Cannot find module '@babel/core')
```
is more helpful than:
```
Failed to load external module @babel/register
```

Reply:
@smialy Sorry, I mistake `console.log(new Error() + '')` for `console.log(new Error())`.
The former does not output stack trace.

Reply:
@smialy Now I think this modification is good, but add/modify a test case to `test/flag-require.js`, please.